### PR TITLE
test: run the Tokenserver E2E tests without a cached JWK

### PR DESCRIPTION
**Pull request by ethowitz**
## Description

Runs the Tokenserver E2E integration tests against a Tokenserver without a cached FxA public JWK. This was done to ensure that we have test coverage for the OAuth verification code paths that involve making a request to FxA to verify tokens. We didn't have coverage before and it hid [a bug](https://github.com/mozilla-services/syncstorage-rs/pull/1389) where we were incorrectly handling `BlockingError`s returned by Actix's `web::block`.

## Testing

I made sure that the `test_unauthorized_oauth_error_status` test failed without ebdd609 and passed with it

### CODE REVIEW

**Review by jrconlin — APPROVED** [review]
Since this came up in a few discussions over on the `syncstorage` issues, I wonder if we want to add a comment describing what values to use here.

**Comment by ethowitz** [comment]
@jrconlin Good call! I've updated the doc comment on Tokenserver's `Settings` struct. Also, I've added a few new tickets to make updates to syncstorage-rs's documentation:
* https://mozilla-hub.atlassian.net/browse/CONSVC-1425 (turn on the `missing_docs` lint, which will require any public item to have a doc comment)
* https://mozilla-hub.atlassian.net/browse/SYNC-3341 (auto build and deploy syncstorage-rs's `cargo docs` on CI, similar to [what we have for Merino](https://mozilla-services.github.io/merino/rustdoc/merino/))
* https://mozilla-hub.atlassian.net/browse/SYNC-3342 (audit and rewrite README, add an ARCHITECTURE.md file)

**Review by jrconlin — DISMISSED** [review]

**Comment by ethowitz** [comment]
Sorry for the noise, had to rebase -- I'll merge this right away after it's re-approved

**Review by pjenvey — APPROVED** [review]

**Pull request merged**
